invenio_app_ils/mappings/v6/document_requests/document_request-v1.0.0.json

```diff
@@ -0,0 +1,61 @@
+{
+  "mappings": {
+    "document_request-v1.0.0": {
+      "date_detection": false,
+      "numeric_detection": false,
+      "properties": {
+        "$schema": {
+          "type": "keyword"
+        },
+        "_created": {
+          "type": "date"
+        },
+        "_updated": {
+          "type": "date"
+        },
+        "authors": {
+          "type": "text"
+        },
+        "cancel_reason": {
+          "type": "text"
+        },
+        "document_pid": {
+          "type": "keyword"
+        },
+        "isbn": {
+          "type": "keyword"
+        },
+        "issn": {
+          "type": "keyword"
+        },
+        "issue": {
+          "type": "keyword"
+        },
+        "note": {
+          "type": "text"
+        },
+        "page": {
+          "type": "keyword"
+        },
+        "patron_pid": {
+          "type": "keyword"
+        },
+        "pid": {
+          "type": "keyword"
+        },
+        "publication_year": {
+          "type": "keyword"
+        },
+        "state": {
+          "type": "keyword"
+        },
+        "title": {
+          "type": "text"
+        },
+        "volume": {
+          "type": "keyword"
```

Comment:
I would go for text here, I would say users will probably input something like `Volume 1`